packages/stack/src/views/Stack/Card.tsx

```diff
@@ -33,6 +34,7 @@ import ModalStatusBarManager from '../ModalStatusBarManager';
 import CardSheet, { CardSheetRef } from './CardSheet';
 
 type Props = ViewProps & {
+  animation?: StackAnimationTypes;
```

Comment:
This prop doesn't seem to be used in this in this component. There are also both `animation` and `styleInterpolator` passed. Only one needs to be passed since `styleInterpolator` can be determined from the `animation` for whichever parent is passing it.